src/composables/userLoginFunction.js

```diff
@@ -0,0 +1,19 @@
+import { signInWithEmailAndPassword } from "firebase/auth"
+import  {appAuth}  from "./firebaseConfig/config"
+
+
+async function UserLogin(emailAddress, password){
+    try {
+    const userInfo = await signInWithEmailAndPassword(appAuth, emailAddress, password)
+    const user = userInfo.user;
+    console.log('Login successful', user)

```

Comment:
Please remove the console.log function In all appearances in the code

Reply:
We could also return a value if the user is signed in. like `true` or `false` so we could use that as a way to route to another page.
So check if the user isn't null || undefined then return true. or if there's a catch return the error

Reply:
Alright